Automattic/wp-calypso
# Google Domains Owners: Add home banner

Related to https://github.com/Automattic/dotcom-forge/issues/3473

## Proposed Changes

We add a banner on the top-carousel.

P2: p9Jlb4-8Df-p2
Figma: mbuq4o5QxohQpBU9YoexXo-fi-1542%3A28718

| Desktop | Mobile |
| --- | --- |
| ![image](https://github.com/Automattic/wp-calypso/assets/402286/809d7376-875e-43a3-83b4-3c37d2a3af57) | ![image](https://github.com/Automattic/wp-calypso/assets/402286/865937dd-759d-4af3-b899-dfca705b44a0) |

## Testing Instructions

* On your sandbox, apply this diff: D119284-code
* As an automattician
* Go to `/home` on a launched site.
* One of the banners should match the screenshot.
* Check impression track event: `calypso_customer_home_card_impression` > card: `home-task-google-domain-owners`
* Check CTA track event: `calypso_customer_home_task_start` > task: `home-task-google-domain-owners`

## Pre-merge Checklist
